components/common/listingCart/shared/ListingCartPriceInput.vue

```diff
@@ -14,6 +14,7 @@
       v-if="check"
       no-shadow
       class="shade-border-color ml-2 height-40"
+      icon-pack="fas"
```

Comment:
can we use `fass` in here? if not, keep it `fas`

Reply:
will keep fas